server/services/odata.service.js

```diff
@@ -256,12 +294,17 @@ module.exports = class ODataService {
 
       console.log(`Patching URL: [${url}]`)
 
-      await fetch(url, {
+      patchCommands.push(fetch(url, {
         method: 'PATCH',
         headers,
         body
-      })
+      }))
     }
+    await Promise.all(patchCommands).then(() => {
+      console.log('All supporting documents patched successfully:')
+    }).catch((error) => {
+      console.log('Error occurred while patching supporting documents', error)
```

Comment:
The application has logging and error handling, we shouldn't need to log to the console or catch the error here